firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TesterApiHttpClient.java

```diff
@@ -104,6 +105,26 @@ JSONObject makeGetRequest(String tag, String path, String token)
    */
   JSONObject makePostRequest(String tag, String path, String token, String requestBody)
       throws FirebaseAppDistributionException {
+    return makePostRequest(tag, path, token, requestBody, new HashMap<>());
+  }
+
+  /**
+   * Make a POST request to the tester API at the given path using a FIS token for auth with
+   * feedback trigger type.
+   *
+   * @return the response body
+   */
+  public JSONObject makePostRequest(
+      String tag, String path, String token, String requestBody, FeedbackTrigger trigger)
```

Comment:
Instead of adding this method, I'd prefer to make `makePostRequest(String tag, String path, String token, String requestBody, Map<String, String> extraHeaders)` below package private, and then call that directly from `FirebaseAppDistributionTesterApiClient#createFeedback()`. That way this this class stays agnostic to feedback-specific logic.